lib/from_openstudio/model.rb

```diff
@@ -281,27 +281,35 @@ def self.schedtypelimits_from_model(openstudio_model)
       result
     end
 
-    # Create HB Schedule Ruleset from OpenStudio Ruleset
-    def self.scheduleruleset_from_model(openstudio_model)
+    def self.schedules_from_model(openstudio_model)
       result = []
       openstudio_model.getScheduleRulesets.each do |sch_ruleset|
         sched_hash = ScheduleRulesetAbridged.from_schedule_ruleset(sch_ruleset)
         $schedules[sched_hash[:identifier]] = sched_hash
         result << sched_hash
       end
-      result
-    end
-
-    # Create HB Schedule Fixed Interval from OpenStudio Schedule Fixed Interval
-    def self.schedulefixedinterval_from_model(openstudio_model)
-      result = []
       # check if it is a leap year 
       is_leap_year = openstudio_model.getYearDescription.isLeapYear
       openstudio_model.getScheduleFixedIntervals.each do |sch_fix_int|
         sched_fixed_hash = ScheduleFixedIntervalAbridged.from_schedule_fixedinterval(sch_fix_int, is_leap_year)
         $schedules[sched_fixed_hash[:identifier]] = sched_fixed_hash
         result << sched_fixed_hash
       end
+      unless $heating_setpoint_schedule.nil?
+        result << $heating_setpoint_schedule
+      end
+      unless $cooling_setpoint_schedule.nil?
+        result << $cooling_setpoint_schedule
+      end
```

Comment:
It would seem that this code is not doing anything right now since this `$heating_setpoint_schedule` variable isn't really created until the Room translation is done, which is happening after the translation of the schedules. Can we just move these 6 lines of code to happen after the room translation in `self.translate_from_openstudio`?

Reply:
Thanks for catching that @chriswmackey.
I have implemented the check for ScheduleRuleset and ScheduleFixedInterval.